.drone.yml

```diff
@@ -2,153 +2,151 @@ kind: pipeline
 name: terraform-google-astronomer-gcp
```

Comment:
We should delete this file. It's not going to be useful to us anymore, and we can always dig into git history if we need to reference it.

Reply:
okay. Let me update the PR